packages/image_picker/image_picker_ios/ios/Classes/FLTPHPickerSaveImageToPathOperation.m

```diff
@@ -184,4 +190,41 @@ - (void)processImage:(NSData *)pickerImageData API_AVAILABLE(ios(14)) {
   }
 }
 
+/**
+ * Processes the video.
+ */
+- (void)processVideo API_AVAILABLE(ios(14)) {
+  NSString *typeIdentifier = self.result.itemProvider.registeredTypeIdentifiers.firstObject;
+  [self.result.itemProvider
+      loadFileRepresentationForTypeIdentifier:typeIdentifier
+                            completionHandler:^(NSURL *_Nullable videoURL,
+                                                NSError *_Nullable error) {
+                              if (error != nil) {
+                                FlutterError *flutterError =
+                                    [FlutterError errorWithCode:@"invalid_image"
+                                                        message:error.localizedDescription
+                                                        details:error.domain];
+                                [self completeOperationWithPath:nil error:flutterError];
+                                return;
+                              }
+                              NSError *saveError;
+                              NSURL *destination =
+                                  [FLTImagePickerPhotoAssetUtil saveVideoFromURL:videoURL
+                                                                           error:&saveError];
+                              if (destination == nil) {
+                                if (!saveError ||
+                                    (saveError && saveError.code != NSFileWriteFileExistsError)) {
```

Comment:
The `saveError` here is redundant, since we can only pass the `||` if it's non-nil.

Reply:
But why is being unable to write over an existing file not an error we want to report?